### ISSUE TITLE
Rarity delimiter units?

### ISSUE THREAD

**Issue by notsellingeth** [HashLips/hashlips_art_engine#1057]
When using the rarity delimiter, is `#50 = 50%?`

Would 5% need to be `#05 or #5`?

Does having some traits with the delimiter make the generation ignore those without?  I can't seem to  make a trait appear only 2% of the time and that is my goal.

**Comment by bolshoytoster** [comment]
@notsellingeth
>  is `#50 = 50%?`

Not necessarily, if all of the rarities in a layer add up to 100 then it acts as a percentage, but usally it's just out of whatever they add up to.

> Would 5% need to be `#05 or #5`?

It doesn't matter.

> Does having some traits with the delimiter make the generation ignore those without?

Yes

> I can't seem to make a trait appear only 2% of the time and that is my goal.

You could make the rarities of the layer add to 100 and make that trait `#2`, or you could make them add up to 50 and make the trait `#1`.

**Comment by Catzskull** [comment]
Trying to set 1of one but seems like it dosent work that way, please hel
<img width="1200" alt="Screen Shot 2022-04-13 at 3 39 27 PM" src="https://user-images.githubusercontent.com/102982983/163259602-db8653ed-d06a-4c08-9b0c-8416c0953b74.png">
p

**Comment by 0xAltar** [comment]
what about 0.5% or something?

**Comment by bolshoytoster** [comment]
@aratarusu that should work.
